conda/core/index.py

```diff
@@ -48,8 +48,11 @@
 
 fail_unknown_host = False
 
+
+REPODATA_PICKLE_VERSION = 1
```

Comment:
This gives us an easy, but manual, way to invalidate pickled data when we know the internal data structures have changed.